API Generator: Generate better API for Many-to-one-relations with `orphanRemoval` activated where the reverse side has its own API generated (#1478)
vivid-planet/comet: .changeset/giant-apples-cheer.md

---
"@comet/cms-api": minor
---

> CRUD Generator: Add the concept of rootArg that can be set in @CrudField for a ManyToOne relation

Reviewer: ```suggestion
API Generator: Add the concept of `rootArg` that can be set in `@CrudField()` for a ManyToOne relation
```

---

> "@comet/cms-api": minor

Author: ```suggestion
"@comet/cms-api": major
```

---

> API Generator: Generate better Api for ManyToOne relations with orphanRemoval activated where the reverse side has its own Api generated.
> 
> - add id as arg to create mutation
> - add id as arg to list query

Reviewer: ```suggestion
API Generator: Generate better API for Many-to-one-relations with `orphanRemoval` activated where the reverse side has its own API generated

- Add `id` as argument to create mutation
- Add `id` as argument to list query
```
